base/linalg/lapack.jl

```diff
@@ -94,6 +100,17 @@ function chkfinite(A::StridedMatrix)
     return true
 end
 
+# LAPACK version number
+function laver()
+    major = BlasInt[0]
+    minor = BlasInt[0]
+    patch = BlasInt[0]
+    ccall((@blasfunc(ilaver_), liblapack), Void,
+        (Ptr{BlasInt}, Ptr{BlasInt}, Ptr{BlasInt}),
+        major, minor, patch)
+    return major[1], minor[1], patch[1]
+end
```

Comment:
The latest version uses `Ref`s, but I've kept the tuple return because, in general, the return from the LAPACK is similar to what is returned from the subroutines. Sometimes we make some postprocessing in the wrappers, but most often this is done down the pipeline and I don't think we return "special" Julia types from any of the LAPACK wrappers.